tests/test_config_consistency.py

```diff
@@ -346,15 +364,25 @@ def test_kafka_integration_enabled_false(self):
 class Test_Config_IntegrationEnabled_True:
     """ Verify behavior of integrations automatic spans """
 
-    def setup_kafka_integration_enabled_true(self):
+    def setup_integration_enabled_true(self):
```

Comment:
nit: Can we have two tests here? One for the dbm endpoint and one for the kafka endpoint. 

We can mark the kafka endpoint as irrelevant for php and mark the new dbm test as irrelevant for not kafka. I think this would make this test easier to read. 

This isn't a blocker I am also fine with the current changes

Reply:
I guess depends on the purpose of the test, if we're testing to see if integrations are enabled/disabled then i'm in favor of keeping as is. If we're testing to see if kafka or pdo specifically is enabled or disabled them i'm in favor of splitting it